js-extensions/src/ExtensionRenderer.jsx

```diff
@@ -1,7 +1,7 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
-var ExtensionStore = require('./ExtensionStore.js').instance;
-var ResourceLoadTracker = require('./ResourceLoadTracker').instance;
+var importedExtensionStore = require('./ExtensionStore.js').instance;
+var importedResourceLoadTracker = require('./ResourceLoadTracker').instance;
```

Comment:
... that's fair, though conversely I feel ill every time I see things instantiated in JS modules and exported essentially as static references. At least this way, you can get a reference to the class for testing/etc. All ears for a better solution that doesn't look like worst practice Java code from 1995.